### PR TITLE
UX-18 Avatar

### PR DESCRIPTION
**Pull request by DanLipsitt**
Implement MLAvatar

### CODE REVIEW

File: src/ml-avatar.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import React from 'react'
+import PropTypes from 'prop-types'
+import { UserOutlined } from '@ant-design/icons'
+
+/**
+ * A default icon is shown unless children or `src` are present.
+ */
+const MLAvatar = (props) => {
+  return (
+    <Avatar icon={props.children || props.src ? null : <UserOutlined />} {...props}>
```
Comment (phoenixeliot) [inline src/ml-avatar.js]: It's not obvious to me what the order of operations is on `props.children || props.src ? null : <UserOutlined />`; could you add some parens to make it clear?

File: stories/18-Avatar.stories.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+export const WithInitials = () => (
+  <MLAvatar size={text('size')}>
+    {text('childText', 'DHS')}
+  </MLAvatar>
+)
+export const WithImage = () => (
+  <MLAvatar
+    size={text('size')}
+    src={text('src', 'https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png')}
```
Comment (phoenixeliot) [inline stories/18-Avatar.stories.js]: `text` knobs should have a second argument for the default string. I think this isn't the kind of knob you want for this, anyway — it should be one of a few options, or a number (or rather, a combo of those two options somehow)

Comment (DanLipsitt) [reply stories/18-Avatar.stories.js]: I was able to specify the union of number and limited text options in PropTypes, but I don't know if Knobs can do it. Entering a number does change the size but I now see it does so in an erroneous way. I'll see if there's a better knob.

Comment (DanLipsitt) [reply stories/18-Avatar.stories.js]: OK, there is no built in knob, but I added integer parsing. Let me know what you think.

**Comment by DanLipsitt** [comment]
Rebased. @Phoenix please also see earlier [discussion](https://github.com/marklogic/design-system/pull/39#discussion_r418404126).

**Review by brucean52 — APPROVED** [review]

**Pull request merged**